packages/cli/lib/projects.js

```diff
@@ -280,6 +281,10 @@ const ensureProjectExists = async (
         return false;
       }
     }
+    if (err.statusCode === 401 && err instanceof HubSpotAuthError) {
```

Comment:
I think you can use `isSpecifiedError` here and pass in the statusCode option. It's a little verbose but is more consistent

Reply:
Ah so that `isSpecifiedError` specifically checks for an error with `err.name === 'StatusCodeError'` but in this case, the error comes through as error name `HubSpotAuthError`

I'll create a new util for these specifically, probably `isHubSpotAuthError` and add that in all the appropriate spots